NuGet.config

```diff
@@ -4,5 +4,6 @@
     <clear />
     <!-- ensure only the sources defined below are used -->
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="dotnet internal feed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" protocolVersion="3" />
```

Comment:
...it never occurred to me that the keys could contain spaces...  TIL.

Reply:
yes, they can :)